src/playground/components/Configuration.js

```diff
@@ -272,6 +261,19 @@ export default function Configuration({
 		return config;
 	};
 
+	const parserValue = parser => {
+		if (
+			(typeof parser === "object" &&
+				parser.meta.name === "typescript-eslint/parser") ||
+			parser === "@typescript-eslint/parser" ||
+			parser === typeScriptESLintParser
```

Comment:
I'm not sure I understand this change. Shouldn't the `Open in Playground` button send `"parser":"@typescript-eslint/parser"` in `options.languageOptions`, not the parser object?

Reply:
I tried `"parser":"@typescript-eslint/parser"` but got this error, and it is not working as expected:

<img width="933" height="62" alt="image" src="https://github.com/user-attachments/assets/d3725d61-0d07-487a-bee9-1f78d03684f3" />

Reply:
Can you share a branch with a code change that is causing this error?

Note that we want to pass `"parser":"@typescript-eslint/parser"` to the playground, not to the code that underlines errors in code example on the docs site.

https://github.com/eslint/eslint/blob/f46fc6c137c951bc73cf3bd9446053c1b11f769b/docs/.eleventy.js#L231-L236

Reply:
Seems you are right, `"parser":"@typescript-eslint/parser"` is working and have implemented in https://github.com/eslint/eslint/pull/19671.

So I think this PR can be closed, thanks.